docs/source/user-guide/guide_api.rst

```diff
@@ -35,42 +24,104 @@ or by running this drush command, passing in the dataset ID:
 
     .. code-block::
 
-      drush dkan:dataset-info [datasetID]
+      drush dkan:dataset-info {datasetID}
 
+In the following examples, we will use {datasetID} to represent the dataset identifier, {distributionID} to
+represent the distribution identifier, and {index} to represent which distribution is being referenced on a
+dataset.
 
-How to set the moderation state through the API.
-------------------------------------------------
+Anonymous API Examples
+----------------------
 
-The available moderation states are: draft, published, hidden, orphaned, and archived.
-Learn more about :term:`Moderation State` here.
+As an anonymous user, the API allows read operations of datasets via a browser (GET requests) or with an HTTP Client.
```

Comment:
Is this needed?

Reply:
Adjusted this to "without authentication" since anonymous user is more of a Drupal term.